### PR TITLE
Implemented page one except for the school searching dropdown

### PR DESCRIPTION
**Pull request by Eric1015**
I have done all the fields in the first page except for the searching schools dropdown.
The content of this PR can be a good reference for other people who are working on page two or three.

The following is the current situation of the first page
image

## Update

Added the autocomplete dropdown for searching the schools and majors
image

### CODE REVIEW

File: components/Signup/PageOne.vue [hunk 1]
```diff
@@ -1,29 +1,236 @@
...
       <b-input v-model.trim="v.hacker.firstname.$model" />
     </b-field>
+
     <b-field
-      label="Last Name"
+      label="What is your Last Name?"
+      class="required"
       :type="v.hacker.lastname.$error ? 'is-danger' : ''"
-      :message="!v.hacker.lastname.required ? 'Required' : ''"
+      :message="v.hacker.firstname.$error ? (!v.hacker.firstname.required ? 'Required' : '') : ''"
```
Comment (ianmah) [inline components/Signup/PageOne.vue]: this should be last name right

File: components/Signup/PageOne.vue [hunk 2]
```diff
@@ -1,29 +1,236 @@
...
+      <b-radio v-model.trim="v.hacker.over19.$model" name="over19" native-value="yes">
+        Yes
+      </b-radio>
+      <b-radio v-model.trim="v.hacker.over19.$model" name="over19" native-value="no">
+        No
+      </b-radio>
+    </b-field>
+
+    <b-field
+      label="What level of education are you currently studying at?"
```
Comment (ianmah) [inline components/Signup/PageOne.vue]: idk if I like this copy

File: components/Signup/PageOne.vue [hunk 3]
```diff
@@ -1,29 +1,236 @@
...
+      <b-select v-model.trim="v.hacker.major.$model" placeholder="Please Select">
+        <option value="computer science">
+          Computer Science
+        </option>
+      </b-select>
+    </b-field>
+
+    <b-field
+      v-if="v.hacker.education === 'undergraduate' || v.hacker.education === 'graduate'"
+      label="What do you plan on studying?"
```
Comment (ianmah) [inline components/Signup/PageOne.vue]: or 'What is your major?' ?

File: components/Signup/PageOne.vue [hunk 4]
```diff
@@ -1,29 +1,236 @@
...
+      </b-select>
+    </b-field>
+
+    <b-field
+      label="What is your graduation year?"
+      class="required"
+      :type="v.hacker.gradyear.$error ? 'is-danger' : ''"
+      :message="v.hacker.gradyear.$error ? (!v.hacker.gradyear.required ? 'Required' : (!v.hacker.gradyear.withinValidRange ? 'Not valid year' : '')) : ''"
+    >
+      <b-numberinput v-model.trim="v.hacker.gradyear.$model" min="2019" max="2027" />
```
Comment (ianmah) [inline components/Signup/PageOne.vue]: I think we agreed to make the minimum lower

File: components/Signup/PageOne.vue (hunk 1 above)
Comment (Eric1015) [reply components/Signup/PageOne.vue]: Oops thank you for pointint out

File: components/Signup/PageOne.vue (hunk 3 above)
Comment (Eric1015) [reply components/Signup/PageOne.vue]: sure

File: components/Signup/PageOne.vue (hunk 4 above)
Comment (Eric1015) [reply components/Signup/PageOne.vue]: Oh forgot to do it

File: components/Signup/PageOne.vue (hunk 2 above)
Comment (Eric1015) [reply components/Signup/PageOne.vue]: Hum, we can discuss this somewhere

**Comment by Eric1015** [comment]
I fixed the <code>TypeError: Cannot read property 'toLowerCase' of undefined</code> error

File: components/Signup/PageOne.vue (hunk 4 above)
Comment (ianmah) [reply components/Signup/PageOne.vue]: How come its still 2019?

**Comment by Eric1015** [comment]
I changed the minimum of the graduation year to 1900

**Pull request merged**